lib/html.rb

```diff
@@ -35,7 +35,10 @@ def self.generate(description)
     File.write(File.join(target, "index.html"), template.render(binding))
     File.write(File.join(target, "assets/description.js"),<<-EOT
       function getDescription() {
-        return JSON.parse('#{description.to_hash.to_json}')
+        return JSON.parse('#{description.to_hash.to_json
+          .gsub("\\\"", "&quote")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer single-quoted strings inside interpolations.